docs/security.md

```diff
@@ -37,8 +37,8 @@ service/binary name and package version. The `APP_ID` takes the form of
       bar:
         start: bin/bar
 
-and the app was uploaded to the `myorigin` origin in the store, then the
-`APP_ID` for the `bar` service is `foo.myorigin_bar_0.1`. The `APP_ID` is used
+and the app was uploaded to the `mydeveloper` developer in the store, then the
+`APP_ID` for the `bar` service is `foo.mydeveloper_bar_0.1`. The `APP_ID` is used
```

Comment:
s/mydeveloper/snapdev/, probably worth as a mechanical change to catch all occurrences.